src/main/java/racing/view/ResultView.java

```diff
@@ -0,0 +1,25 @@
+package racing.view;
+
+import racing.domain.Cars;
+
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class ResultView {
+    private static final String POSITION = "-";
+
+    private ResultView() {
+    }
+
+    public static void printCarPosition(Cars cars) {
+        cars.getCarList().stream().forEach(car -> {
+            System.out.println(repeat(car.getPosition()));
+        });
```

Comment:
```suggestion
       cars.getCarList().forEach(car -> System.out.println(repeat(car.getPosition())));
```
stream forEach 를 사용하지 않아도 될거 같아요! 😄